src/calculateRentalCost.js

```diff
@@ -29,6 +29,19 @@
 
 function calculateRentalCost(days) {
   // write code here
-}
+  const dayPrice = 40;
+  const sum = days * dayPrice;
+  const countDay0 = 0;
+  const countDay3 = 3;
+  const countDay7 = 7;
+
+  if (days > countDay0 && days < countDay3) {
+    return sum;
+  } else if (days >= countDay3 && days < countDay7) {
+    return sum - 20;
```

Comment:
this is considered as magic numbers we well. Try to come up with meaningful name for it. It can be shortStayDiscount or something like this